### PR TITLE
Feature login implementation

### PR DESCRIPTION
**Pull request by HarishChoure**
Linked Work Items

https://expense-management-system-batch-1-2022.atlassian.net/browse/EMSB12-77
Checklist

Checklist

 Code adheres to [Git and Pull Request Guidelines]
 Code adheres to [Angular Guidelines]
 Code adheres to [Front-end Checklist]

### CODE REVIEW

File: src/app/user-sign-up/login/login.component.ts [hunk 1]
```diff
@@ -10,31 +14,84 @@ import { Router } from '@angular/router';
...
     return this.loginForm.get('password');
   }
 
-  // Form submit method
-  onSubmit(): void{
+  /**
+   * login method are used to subscribe data
+   * @param existing_user store the data of the user credential
+   *  */
+  userLogin(existing_user: UserLogin)  {
```
Comment (AshaPatel216) [inline src/app/user-sign-up/login/login.component.ts]: Follow variable naming conventions. existing_user -> existing-user

File: src/app/user-sign-up/login/login.component.ts [hunk 2]
```diff
@@ -10,31 +14,84 @@ import { Router } from '@angular/router';
...
+    this.loggedUSer.login(existing_user).subscribe(
+      (response) => {
+        console.log(response);
```
Comment (AshaPatel216) [inline src/app/user-sign-up/login/login.component.ts]: Remove console log

File: src/app/user-sign-up/login/login.component.ts [hunk 3]
```diff
@@ -10,31 +14,84 @@ import { Router } from '@angular/router';
...
+    const user: UserLogin = {
+      userNameOrEmailAddress: formData.email,
+      password: formData.password,
+      rememberMe: true,
+    }
     this.loginForm.reset();
-    this.router.navigate(['dashboard']);
+    //userLogin method are used to subscribe data
+    //@param user store the user credentail data
+    this.userLogin(user)
```
Comment (AshaPatel216) [inline src/app/user-sign-up/login/login.component.ts]: Semi colon

File: src/environments/environment.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+// import { Environment } from "@abp/ng.core";
```
Comment (AshaPatel216) [inline src/environments/environment.ts]: Commented code

File: src/app/user-sign-up/login/login.component.ts (hunk 1 above)
Comment (HarishChoure) [reply src/app/user-sign-up/login/login.component.ts]: Ok dii

File: src/environments/environment.ts (hunk 4 above)
Comment (HarishChoure) [reply src/environments/environment.ts]: Di I removed all the commented code

**Review by AshaPatel216 — APPROVED** [review]

**Pull request merged**